src/axiosConfig/axiosInstance.ts

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable prefer-const */
 import axios from "axios";
 
 let axiosInstance = axios.create({
-  baseURL: "https://upskilling-egypt.com:3000/api/v1",
+  baseURL: "https://upskilling-egypt.com:3000/api/v0",
```

Comment:
I suggest creating .env file to store the base URL. This will be better if one of our endpoints depends upon a paid API. This means that we are making it available for everyone for free and we pay the bill.